### ISSUE TITLE
Secondary Font (Font Awesome)

### ISSUE THREAD

**Issue by alindesign** [felixse/FluentTerminal#107]
Hi there, 
It's there a possible way to add a secondary font for icons, for example, right now [FontAwesome](https://fontawesome.com/) icons (inserted as unicode) are not showing on it.

My bash configuration on:
- [Hyper](https://github.com/zeit/hyper)
![image](https://user-images.githubusercontent.com/12038126/47609691-349eeb80-da4c-11e8-844a-333e5a3650c8.png)

- FluentTerminal
![image](https://user-images.githubusercontent.com/12038126/47609703-687a1100-da4c-11e8-8135-305af4760786.png)

**Comment by felixse** [comment]
Looks like you are using a patched nerd font for Hyper. Have you configured the same font for Fluent Terminal?

**Comment by alindesign** [comment]
Hi @felixse, thanks for response, well on the Hyper i'm using multiple fonts (since it's a web app).
![image](https://user-images.githubusercontent.com/12038126/47616781-0b24a500-daca-11e8-84e7-70afdbef7e12.png)

But on FluentTerminal, i can select just 1 font, so no patched fonts ...

**Comment by felixse** [comment]
Did you try installing one of these: https://nerdfonts.com/ ?

**Comment by alindesign** [comment]
I'm trying it right now, i just founded :D

**Comment by alindesign** [comment]
@felixse Seems to be good with that one, icons look little small but it's fine,
![image](https://user-images.githubusercontent.com/12038126/47617005-2db7bd80-dacc-11e8-8e14-a3a280ee1b26.png)

But is that possible to add fallback or multiple fonts right now?

**Comment by felixse** [comment]
There is nothing implemented right now, but in theory we should be able to pass a concatenation the same as hyper does. We just would need to implement some UI for this.

**Comment by alindesign** [comment]
Nice, thanks 😊

**Comment by spencerwooo** [comment]
Hi, I just wanted to ask if there's any progress on adding a fallback font? I reallllllly want this feature. ❤❤❤

**Comment by felixse** [comment]
Not really, I am afraid I don't have any spare time right now to work on this project 😢
